src/runtime/virtcontainers/clh.go

```diff
@@ -1207,7 +1207,7 @@ func (clh *cloudHypervisor) cleanupVM(force bool) error {
 	}
 
 	// Cleanup vm path
-	dir := filepath.Join(clh.store.RunVMStoragePath(), clh.id)
+	dir := filepath.Join(clh.vmstorepath, clh.id)
```

Comment:
If I remember right there two dirs. The run and persist path. Are keeping the two seperate?

Reply:
For now I was keeping "the same".  IIRC, the run store one is for storage. ideally we won't have 2 paths...